lib/beaker/host/unix/exec.rb

```diff
@@ -302,7 +304,7 @@ def ssh_permit_user_environment
       directory = tmpdir
       exec(Beaker::Command.new("echo 'PermitUserEnvironment yes' | cat - /etc/ssh/sshd_config > #{directory}/sshd_config.permit"))
       exec(Beaker::Command.new("mv #{directory}/sshd_config.permit /etc/ssh/sshd_config"))
-      exec(Beaker::Command.new("echo '' >/etc/environment")) if /ubuntu-2(0|2).04/.match?(self['platform'])
+      exec(Beaker::Command.new("echo '' >/etc/environment")) if /ubuntu-2(0|2|4).04/.match?(self['platform'])
```

Comment:
@skyamgarp does it make sense to match for all ubuntus here? (and why is that even required at all?)

Reply:
This is to add puppet to `~/.ssh/environment`. Without this change we get error as `puppet: command not found`. I am not sure how does this work exactly for earlier versions.

Reply:
Is there a chance you can test that on other Ubuntu platforms you've to support? Then we could remove the version check and run it for every ubuntu. So we don't have to update it for ubuntu 26.04 again.